product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfigCompositeApplication/pom.xml

```diff
@@ -342,6 +343,18 @@
       <version>1.0.0</version>
       <type>xml</type>
     </dependency>
+    <dependency>
+      <groupId>org.wso2.ei.product-scenarios.scenario_01.proxy-service</groupId>
+      <artifactId>1_6_10_2_Proxy_alterPayloadWithInlineJavaScript</artifactId>
+      <version>1.0.0</version>
+      <type>xml</type>
+    </dependency>
```

Comment:
we do not need to add this dependency. we have already added
<dependency>
      <groupId>org.wso2.ei.product-scenarios.scenario_01.proxy-service</groupId>
      <artifactId>1_6_10_2_Proxy_AlterPayloadWithInlineJavaScript</artifactId>
      <version>1.0.0</version>
      <type>xml</type>
</dependency>